lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -347,10 +351,10 @@ def inner(self, default_store, module_count, mongo_calls, sql_queries, *args, **
         return inner
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 3, 4, 21),
-        (ModuleStoreEnum.Type.mongo, 20, 4, 21),
-        (ModuleStoreEnum.Type.split, 3, 13, 21),
-        (ModuleStoreEnum.Type.split, 20, 13, 21),
+        (ModuleStoreEnum.Type.mongo, 3, 4, 22),
```

Comment:
@benpatterson or @ormsbee do you know what the performance cost is of an sql query, and wether or not we should try to maintain currently query levels by looking for unnecessary existing queries? For this feature, I am going to make it so that this additional sql query only happens for discussion topic IDs that are linked to teams (we can pass that context back from the comment service at no additional cost).